### ISSUE TITLE
Nifty future

### ISSUE THREAD

**Issue by illuminate3** [kJamesy/NiftyCMS#1]
Hello,

Just wondering what you are planning to do with Nifty.
Do you plan a Laravel 5/5.1 version?

**Comment by kJamesy** [comment]
Yes actually there is a Laravel 5 version: https://github.com/ACWLtd/Nifty
